api/src/main/java/org/pmiops/workbench/rdr/RdrExportEnums.java

```diff
@@ -3,6 +3,7 @@
 import com.google.common.collect.BiMap;
 import com.google.common.collect.ImmutableBiMap;
 import org.pmiops.workbench.model.*;
```

Comment:
This is existing, but there's a star import here and we're trying to not use those, could you driveby fix?